src/applications/discharge-wizard/helpers/index.jsx

```diff
@@ -263,12 +263,15 @@ export const deriveIsAirForceAFRBAPortal = formValues =>
 
 // v2 Helpers
 
+// Changes Marine Corps discharge board to Navy for naming on results pages.
 export const determineBranchOfService = key =>
   key === RESPONSES.MARINE_CORPS ? RESPONSES.NAVY : key;
 
+// Determines old discharges
```

Comment:
What does this mean? It seems like, based on the utility, it's determining how old the previous discharge upgrade/application is?